docs/docs/guides/integrate/login/hosted-login.mdx

```diff
@@ -177,22 +180,45 @@ Your contributions will play a crucial role in shaping the future of our login s
-
-1. [Create a service user](https://zitadel.com/docs/guides/integrate/service-users/personal-access-token#create-a-service-user-with-a-pat) with a PAT in your instance
-2. Give the user IAM_LOGIN_CLIENT Permissions in the default settings (YOUR_DOMAIN/ui/console/instance?id=organizations)
-   Note: [Zitadel Manager Guide](https://zitadel.com/docs/guides/manage/console/managers)
-3. Deploy login to Vercel: You can do so, be directly clicking the [“Deploy” button](https://github.com/zitadel/typescript?tab=readme-ov-file#deploy-to-vercel) at the bottom of the readme in our [repository](https://github.com/zitadel/typescript)
-4. If you have used the deploy button in the steps before, you will automatically be asked for this step. Enter the environment variables in Vercel
-   - PAT
-   - ZITADEL_API_URL (Example: https://my-domain.zitadel.cloud, no trailing slash)
-5. Add the domain where your login UI is hosted to the [trusted domains](https://zitadel.com/docs/apis/resources/admin/admin-service-add-instance-trusted-domain) in Zitadel. (Example: my-new-zitadel-login.vercel.app)
-6. Use the new login in your application. You have three different options on how to achieve this
-   1. Enable the new login on your application configuration and add the URL of your login UI, with that settings Zitadel will automatically redirect you to the new login if you call the old one.
-      ![Login V2 Application Configuration](/img/guides/integrate/login/login-v2-app-config.png)
-   2. Enable the [loginV2 feature](https://zitadel.com/docs/apis/resources/feature_service_v2/feature-service-set-instance-features) on the instance and add the URL of your login. If you enable this feature, the login will be used for every application configured in your Zitadel instance. (Example: https://my-new-zitadel-login.vercel.app)
-   3. Change the issuer in the code of your application to the new domain of your login
-7. Enforce users to have their email verified. By setting `EMAIL_VERIFICATION` to `true` in your environment variables, your users will be enforced to verify their email address before they can log in.
+To try out the new login without modifying it, the easiest way is to just activate it on your Zitadel Cloud instance domain at `https://<YOUR_ZITADEL_INSTANCE_DOMAIN>/ui/v2/login`.
+
+The easiest way to actually customizing it is to fork the https://github.com/zitadel/typescript repo and use the "Deploy" button to run your code on Vercel.
+
+:::caution
+Before you reconfigure Zitadel, we recommend you to create a service user with a personal access token and the IAM_OWNER role for a lockout-failsafe.
+:::
+
+<Tabs queryString="deployment">
+    <TabItem value="zitadel_cloud_standard" label="Zitadel Cloud Standard V2 Login" default="true">
+
+        Zitadel Cloud provides the standard V2 login at the fixed path `/ui/v2/login`. To activate it in your Zitadel Cloud instance, you can follow one of these steps:
```

Comment:
I guess your intention with the "standard" is to tell that it is not customized. In my opinion hosted login is always about our login version. The word standard leads to more confusion. 

```suggestion
        Zitadel Cloud provides the V2 login at the fixed path `/ui/v2/login`. To activate it in your Zitadel Cloud instance, you can follow one of these steps:
```